cql3/statements/select_statement.cc

```diff
@@ -333,7 +333,8 @@ select_statement::do_execute(service::storage_proxy& proxy,
             tracing::make_trace_info(state.get_trace_state()),
             utils::UUID(),
             query::is_first_page::no,
-            options.get_timestamp(state));
+            options.get_timestamp(state),
+            raw_cql_statement);
```

Comment:
Where are you getting this from? I don't see this defined anywhere here.

Reply:
It is from cql_statement.hh file, base class cql_statement.